fix: incorporate dev changes

raphael-jms opened the pull request:
* backup

* sim fix

* feat: fixed some paths

---------

raphael-jms committed:
fix: incorporate dev changes

* backup

* sim fix

* feat: fixed some paths

---------

Co-authored-by: Pedro-Roque <padr@kth.se>

raphael-jms committed:
resolve merge conflict

Pedro-Roque commented:
@raphael-jms rebase from main to make sure the commit "fix: incorporate dev changes" is already merged in your dev.

Pedro-Roque committed:
fix: minor changes

Pedro-Roque committed:
feat: added inertial properties

Pedro-Roque committed:
fix: tab

Pedro-Roque committed:
fix: final tab fix?

Pedro-Roque committed:
fix: now?

Pedro-Roque committed:
fix: sampling time symbol

Pedro-Roque committed:
add: d value

raphael-jms committed:
Update InertialProperties.md

First matrix was not rendered; fix

raphael-jms committed:
Update README.md

Added backup link to the video, as video playback directly in Github Readme is sometimes a bit unreliable
